apps/frontend/src/libs/components/header/libs/components/header-user-info/header-user-info.tsx

```diff
@@ -0,0 +1,31 @@
+import defaultAvatar from "~/assets/img/default-avatar.png";
+import { AppRoute } from "~/libs/enums/enums.js";
+import { useCallback, useNavigate } from "~/libs/hooks/hooks.js";
+import { type UserDto } from "~/modules/users/users.js";
+
+import styles from "./styles.module.css";
+
+type Properties = {
+	user: UserDto;
+};
+
+const HeaderUserInfo: React.FC<Properties> = ({ user }: Properties) => {
+	const navigate = useNavigate();
+
+	const handleClick = useCallback(() => {
+		navigate(AppRoute.PROFILE);
+	}, [navigate]);
+
+	return (
+		<button className={styles["container"]} onClick={handleClick}>
+			<img
+				alt={`${user.name}'s avatar`}
+				className={styles["avatar"]}
+				src={user.avatarUrl ?? defaultAvatar}
+			/>
+			<span className={styles["username"]}>{user.name}</span>
+		</button>
```

Comment:
```suggestion
		<NavLink className={styles["container"]} to={AppRoute.PROFILE}>
			<img
				alt={`${user.name}'s avatar`}
				className={styles["avatar"]}
				src={user.avatarUrl ?? defaultAvatar}
			/>
			<span className={styles["username"]}>{user.name}</span>
		</NavLink>
```

Reply:
Can we use `<NavLink>` by direct import from "react-router-dom"?

Reply:
You should add export from "react-router-dom" in `~libs/components/components.ts`, and import it from components.

Reply:
I think I will wait coach @fshabanov response about it, since it will conflict with existing `<Link>` component in `~libs/components/components.js`

Reply:
Why not use existing Link component?

Reply:
Link component styled for sidebar link

Reply:
Link component should be a global one, not only for sidebar